src/main/resources/application.conf

```diff
@@ -33,6 +33,11 @@ etc-client {
     dao-fork-block-hash = "94365e3a8c0b35089c1d1195081fe7489b528a84b22199c916180db8b28ade7f"
   }
 
+  fast-sync {
```

Comment:
I know this is a WIP but take a look at this https://github.com/ethereumproject/go-ethereum/blob/master/eth/downloader/downloader.go#L43

```go
MaxHashFetch    = 512 // Amount of hashes to be fetched per retrieval request
MaxBlockFetch   = 128 // Amount of blocks to be fetched per retrieval request
MaxHeaderFetch  = 192 // Amount of block headers to be fetched per retrieval request
MaxSkeletonSize = 128 // Number of header fetches to need for a skeleton assembly
MaxBodyFetch    = 128 // Amount of block bodies to be fetched per retrieval request
MaxReceiptFetch = 256 // Amount of transaction receipts to allow fetching per request
MaxStateFetch   = 384 // Amount of node state values to allow fetching per request
```